.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,44 @@
+---
+name: 'Report a bug'
+about: Report a bug 🪲
+title: "[bug]: bug title here"
+labels: bug
+assignees: ''
+
+---
+
+**What did you do**
+Please provide steps for us to reproduce this issue? Example for config file are very welcomed!
+
+**What did you expect to see?**
+
+
+**What did you see instead? Under which circumstances?**
+
+
+**Are you currently working around this issue?**
+How are you currently solving this problem?
+
+**Additional context**
+Anything else we should know?
+
+**Attachments**
+If you think you might have additional information that you'd like to include via an attachment, please do - we'll take a look. (Remember to remove any personally-identifiable information.)
+
+**Environment**
+Cloudwatch version: `X.X.X`
+Platform (linux / windows / arm): `XXX`
+Running using docker / Java?
+
+**Exporter configuration file**
+<!-- Remember to clear private information -->
+
+```yaml
+
+```
+
```

Comment:
Do we want to have the (potentially long) files folded up like this?

<details><summary>configuration file</summary>

```yaml
some: yaml
```

</details>

Reply:
Good idea!